tests/test_parser.py

```diff
@@ -257,6 +259,67 @@ def test_convert_to_date_cast_result_to_date(self, mocked_locale,
                          expected_output)
 
 
+class TestParseParentPage(unittest.TestCase):
+    @patch('smarsy.parse.BeautifulSoup')
+    def test_parent_page_content_called_with_expected_html(self,
+                                                           mocked_soup):
+        html = '<tr></tr>'
+        parent_page_content_to_object(html)
+        mocked_soup.assert_called_with(html, 'html.parser')
+
+    @patch('bs4.BeautifulSoup', side_effect=ValueError)
+    def test_parent_page_content_raise_exeption_with_wrong_file_format(
+            self, mocked_soup):
+        html = 12344
+        self.assertRaises(ValueError, parent_page_content_to_object, html)
+
+    @patch('bs4.BeautifulSoup')
+    def test_bs_safeGet_called_with_expected_tag(self, mocked_soup):
+        soup = mocked_soup('some html', 'html.parser')
+        selector = 'sdfs'
```

Comment:
kind of advice that if you need to pass something that is incorrect, it is better to indicate it somehow. Instead of using some magic characters sequence it is better to have something like 'not  valid html document'